### PR TITLE
Add poetry reading sessions

### PR DESCRIPTION
**Pull request by michelmansour**
- Add server routes to support poetry sessions
- Implement basic client-side session functionality, including
  reordering poems within a session
- Refactor some React components to better reflect application structure

### CODE REVIEW

**Comment by michelmansour** [comment]
@drewrey Please review

**Comment by michelmansour** [comment]
The next step will be to enable moving poems between the main queue and the active session.

File: client/src/ActiveSessionPoemListContainer.jsx [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+
+  componentDidMount = () => {
+    this.loadActiveSession();
+    this.state.intervalId = setInterval(this.loadActiveSession, this.props.route.pollInterval);
+    const el = document.getElementById('activeSessionPoemList');
+    Sortable.create(el, {
+      onUpdate: (evt) => {
+        const reordering = this.state.session.poems.slice();
+        const movedPoem = reordering.splice(evt.oldIndex, 1);
+        reordering.splice(evt.newIndex, 0, ...movedPoem);
```
Comment (drewrey) [inline client/src/ActiveSessionPoemListContainer.jsx:25]: Can we expect `movedPoem` to always be a singleton? If so, is the `...` syntax appropriate here?

**Review by drewrey — APPROVED** [review]

File: client/src/ActiveSessionPoemListContainer.jsx (hunk 1 above)
Comment (michelmansour) [reply client/src/ActiveSessionPoemListContainer.jsx:25]: We probably want to ensure that exactly one poem is moved. But `Array.splice` returns an array, so it's either `...movedPoem` or `movedPoem[0]`. This seems better given any lack of checking (ie, what if `moedPoems` is empty?)

**Pull request merged**